_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -306,4 +367,14 @@ if {[string first "--enable-debug" ${configure.args}] > -1} {
 
 default build.dir {${configure.dir}}
 
-default build.post_args {VERBOSE=ON}
+pre-build {
+    if {${cmake.generator} eq "Ninja"} {
+        if {![tbool use_parallel_build]} {
+            # ninja builds in parallel mode by default
+            build.post_args-append -j1
+        } elseif {${build.jobs} >= 1} {
+            build.post_args-append -j${build.jobs}
+        }
```

Comment:
Ah. I checked how make was called when !use_parallel_build but didn't realise build.jobs was guaranteed to be 1 in that case.